plasmapy/physics/quantum.py

```diff
@@ -281,3 +284,157 @@ def Thomas_Fermi_length(n_e):
     energy_F = Fermi_energy(n_e)
     lambda_TF = np.sqrt(2 * eps0 * energy_F / (3 * n_e * e ** 2))
     return lambda_TF.to(units.m)
+
+
+@check_quantity({
+    'n_e': {'units': units.m**-3, 'can_be_negative': False}
+})
+def Wigner_Seitz_radius(n):
+    r"""Calculate the Wigner-Seitz radius, which approximates the inter-
+    particle spacing. It is the radius of a sphere whose volume is 
+    equal to the mean volume per atom in a solid. This parameter is
+    often used to calculate the coupling parameter.
+    When ion density is used, this is the ion sphere radius.
```

Comment:
Providing the undergrad perspective here! What's the ion sphere?

Reply:
Ions are highly protective of their airspace. They like their personal space so they keep other ions at arm's length (or ion sphere radius) away.

It's just the mean space occupied by a single ion for a particular density. Higher density means less space for each ion.

Reply:
Ah. I see, thanks for the explanation!